extensions.json

```
@@ -874,7 +879,12 @@
   "ms-vscode.vscode-js-profile-table": {
     "repository": "https://github.com/microsoft/vscode-js-profile-visualizer",
     "location": "packages/vscode-js-profile-table",
-    "custom": "npm install && npm run compile && cd packages/vscode-js-profile-table && npx vsce package --yarn --out extension.vsix" 
+    "custom": [
+      "npm install",
+      "npm run compile",
+      "cd packages/vscode-js-profile-table",
+      "npx vsce package --yarn --out extension.vsix"
```

If you were to reset the cwd for each command (which I'd recommend you to do), you'd have to chain these two commands with a `&&`:

```suggestion
      "cd packages/vscode-js-profile-table && npx vsce package --yarn --out extension.vsix"
```

---

Good catch, thank you!